### ISSUE TITLE
Deploying Issue - Was successful at first, went back to comment out .gitignore file

### ISSUE THREAD

**Issue by matthewgsommers** [ga-wdi-boston/game-project#315]
I deployed successfully, realized that I missed the section in which I should comment out the 2 lines in the .gitignore file (as indicated by the deployment page), so I went back and did that,  and now I'm having trouble viewing my page. Its displaying my scss instead of my css even though I commented out all the scss and the scss file isnt even linked.

**Comment by raq929** [comment]
Webpack links the scss for you automatically.
Are you linking  the css in the head of the html?

**Comment by matthewgsommers** [comment]
Yes, with       <link rel="stylesheet" type="text/css" href="assets/styles/index.css">

**Comment by matthewgsommers** [comment]
```
  <link rel="stylesheet" type="text/css" href="assets/styles/index.css">
```

**Comment by matthewgsommers** [comment]
I actually just tried it again and now its working fine... Which is super strange because I didnt edit anything. Maybe it was caused by a time delay? Also, should I only be able to see my page on the "gh-pages" branch? Because I can currently see it on both that and the master branch.

**Comment by matthewgsommers** [comment]
And now the same problem is back. Its displaying the original page again (i.e. the wrong one). I'm really confused on whats happening because I havent changed any of the code in between these two messages.

**Comment by BenGitsCode** [comment]
You're viewing these changes on your deployed gh pages url?

**Comment by matthewgsommers** [comment]
Correct. Via the setting tab on my project.

**Comment by BenGitsCode** [comment]
You mean the settings tab is where you got the github.io link right? And that deployed version isn't showing the correct style changes from css?

**Comment by matthewgsommers** [comment]
Yes, correct on both fronts. The settings tab on my project repo. Correct, my css is being over ridden by the scss even though I have commented out all the scss and didnt even link it in the index.html file. When I refresh the page I see my css layout for a split second and then the scss layout takes over.

**Comment by RealWeeks** [comment]
STOP

**Comment by RealWeeks** [comment]
DON'T DO THIS `<link rel="stylesheet" type="text/css" href="assets/styles/index.css">`

SCSS is just compiled CSS, which means you can write CSS in your .scss file. We're using something call webpack which bundles files for us and releases them at certain times to the browser. By not writing your CSS in the correct place you could be causing it not to load correctly.

**Comment by RealWeeks** [comment]
summary write your css in you .scss file, and don't like additional files... this isn't your grandmas coding.  We're using developer tooling so traditional loading may cause problems.

**Comment by RealWeeks** [comment]
Once done with this repush to your `gh-pages` branch and we'll continue to debug.

**Comment by matthewgsommers** [comment]
I removed all of the scss in an attempt to make the page look like it should locally (styled with my css) but nothing has changed.

**Comment by raq929** [comment]
I would follow Jason's instructions above. Move your css into index.scss.

**Comment by RealWeeks** [comment]
So the problem now is that your local page doesn't match your deployed page?

Post your html and scss

**Comment by RealWeeks** [comment]
also, did you build?

**Comment by matthewgsommers** [comment]
HTML: <!DOCTYPE html>
<html>
    <head>
      <title>API Token Auth</title>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <script src="vendor.bundle.js" type="text/javascript" charset="utf-8" defer></script>
      <script src="bundle.js" type="text/javascript" charset="utf-8" defer></script>
      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
    </head>

```
<body class="container-fluid">
      <div class="container">
          <div class="row">
            <div class="col-md-4 box top-left"></div>
            <div class="col-md-4 box top-center"></div>
            <div class="col-md-4 box top-right"></div>
          </div>
          <div class="row">
            <div class="col-md-4 box middle-left"></div>
            <div class="col-md-4 box middle-center"></div>
            <div class="col-md-4 box middle-right"></div>
          </div>
          <div class="row">
            <div class="col-md-4 box bottom-left"></div>
            <div class="col-md-4 box bottom-center"></div>
            <div class="col-md-4 box bottom-right"></div>
          </div>
        </div>

      <script src="assets/scripts/main.js"></script>
    <div class="displayWinner">
    </div>
    <br>
        <input id="button" type="button" onclick="resetBoard()" name="resetButton" value="Click Here to Reset the Board">
</body>
```

</html>

**Comment by matthewgsommers** [comment]
index.scss: 

$icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
@import '~bootstrap-sass/assets/stylesheets/bootstrap/';

.col-md-4.box {
  border: 2px solid grey;
  color: blue;
  font-size: 24px;
  height: 100px;
  text-align: center;
  width: 100px;
}

.row {
  display: flex;
}

body {
  width: 500px;
  height: 500px;
  background-color: yellow;
}

.container-fluid {
  padding-left: 50px;
}

**Comment by matthewgsommers** [comment]
Also, the following info was in the HTML's head tag, not sure why it didnt paste in my other comment. 

```
  <title>API Token Auth</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <script src="vendor.bundle.js" type="text/javascript" charset="utf-8" defer></script>
  <script src="bundle.js" type="text/javascript" charset="utf-8" defer></script>
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

**Comment by RealWeeks** [comment]
did you put this in yourself? `<script src="assets/scripts/main.js"></script>`

**Comment by RealWeeks** [comment]
Note: In CSS order matters, the last thing you write will overwrite previously written styles.

Also, don't target bootstrap classes.. so this `.col-md-4.box {}` can be this `.box{}` because they are targeting the same thing.

**Comment by gaand** [comment]
Also, this shouldn't be in your `index.html` using our template:

``` html
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

It's not in the `index.html` from the `api-token-auth` talk.

**Comment by RealWeeks** [comment]
but my number 1 question

## Did you Build?

**Comment by matthewgsommers** [comment]
The reason I have those two lines of code you called out is because I started building the game's js/html/css in a separate file than the template because I was encountering linter error from yesterday. I was trying to make progress in a different directory while my linter error got squared away and then I planned to move the code from the other file into the template.

**Comment by RealWeeks** [comment]
That's fine, it's breaking your code. Remove them. Once you do build and let me know what happens.

**Comment by matthewgsommers** [comment]
OK - So my CSS is being deployed correctly now. However I'm a bit confused as to why I removed the link to my main.js file - now my page isnt including any game functionality... Should I have not used the api-token-auth file as my template?

**Comment by RealWeeks** [comment]
You should but that line wasn't in there

**Comment by RealWeeks** [comment]
We're using a tool call webpack which bundles your files for you thus `bundle.js` and `vendor.bundle.js` when you build it puts your code in those files, you can see they are already linked in `index.html`

Not your grandmas coding...

**Comment by RealWeeks** [comment]
I'm gonna take a look at this
